### DISCUSSION TITLE
scikit-HEP installation

### DISCUSSION THREAD

**Discussion by kavita2275** [scikit-hep/scikit-hep#289] [Q&A] [answered]
Hi, 
I have installed scikit-Hep using

 "**conda install -c conda-forge scikit-hep** "

when I try to run the following command in jupyter notebook:

"**from skhep.visual import MplPlotter as skh_plt**"

I get error : "**No module named 'skhep.visual'**

I am not finding any command online to install 'skhep.visual' separately.  

Can you help me in resolving this issue?

**Comment by matthewfeickert** [comment]
@kavita2275 the `skhep.visual` API was removed in PR #151 back in 2021. Can you tell us where you're finding documentation that is recommending the use of this API?

**Reply by kavita2275** [reply]
I was searching for online packages on the Bayesian block. So I found the following link,

https://notebook.community/scikit-hep/scikit-hep/doc/examples/modeling/bayesian_blocks

Is it possible to use Bayesian block using scikit-HEP?

**Comment by kavita2275** [comment]
Thank you for your response!
I was searching for online packages for Bayesian block. So I found the
following link,

https://notebook.community/scikit-hep/scikit-hep/doc/examples/modeling/bayesian_blocks

Is it possible to use Bayesian block using scikit-HEP?


Thanks

**Reply by matthewfeickert** [reply, marked as answer]
@kavita2275 https://github.com/scikit-hep/hepstats (which is bundeled as part of the `scikit-hep` metapackage) includes the Bayesian Block algorithm that is from [a (similar) paper](https://arxiv.org/abs/1207.5578) to the one the link you gave is referencing.

If you have questions on it I'd recommend you ask on the [`hepstats` GitHub Discussions page](https://github.com/scikit-hep/hepstats/discussions/categories/q-a).
